pkg/crc/network/types.go

```diff
@@ -23,27 +23,35 @@ type ResolvFileValues struct {
 type Mode string
 
 const (
-	DefaultMode Mode = "default"
-	VSockMode   Mode = "vsock"
+	SystemNetworkingMode Mode = "system"
+	UserNetworkingMode   Mode = "user"
```

Comment:
usermode would have made more sense

Reply:
I opted out of `usermode` as `crc config set network-mode usermode` felt redundant.
I've renamed the options to `vsock`/`bridged` as was discussed earlier in this PR, code is at https://github.com/cfergeau/crc/tree/network-mode I can file a PR if this naming is preferred.